src/main/java/org/folio/rest/impl/PostGobiOrdersHelper.java

```diff
@@ -140,13 +140,7 @@ public PostGobiOrdersHelper(HttpClientInterface httpClient,
         .build());
       mappings.put(Field.USER_LIMIT, DataSource.builder()
         .withFrom("//PurchaseOption/Code")
-        .withTranslation(s -> {
-          if (s != null) {
-            return CompletableFuture.completedFuture(3);
-          } else {
-            return CompletableFuture.completedFuture(null);
-          }
-        })
+        .withTranslation(this::getPurchaseOptionCode)
```

Comment:
This should probably be pulled out of here... I never intended for it to get checked-in to begin with.  I special-cased it just to put "something" in this field.  [MODGOBI-11](https://issues.folio.org/browse/MODGOBI-11) will need to implement a Mapper function that parses the PurchaseOption Code and return the numeric user limit.  This translation method should be specified just like other translators.

Reply:
I have something to that effect, though not exactly right yet.  I extracted that logic into the method "getPurchaseOptionCode" and listed as an option for translation in the json schema.  So, is this work part of this user story? I only touched it because it had to do with the schema.

Reply:
You can just leave it as-is for now and well tackle it in MODGOBI-11